src/bp/core/server.ts

```diff
@@ -280,10 +280,14 @@ export default class HTTPServer {
     this.app.use(`${BASE_API_PATH}/bots/:botId`, this.botsRouter.router)
     this.app.use(`/s`, this.shortlinksRouter.router)
 
-    this.app.use(function handleErrors(err, req, res, next) {
+    this.app.use((err, req, res, next) => {
```

Comment:
I think the name was there for better details in stack traces, instead of seeing "anonymous function"